config/certmanager/kustomization.yaml

```diff
@@ -0,0 +1,5 @@
+resources:
+- certificate.yaml
+
+configurations:
```

Comment:
do you have a link on how this works ? 

i didnt find configuration section in : https://kubectl.docs.kubernetes.io/references/kustomize/kustomization/